src/components/generic/Alert.tsx

```diff
@@ -1,81 +1,134 @@
-import classes from './Alert.module.scss'
+import React, { createContext, useContext, useState, useEffect, useRef, ReactNode } from 'react';
+import { CheckCircle, Info, Warning, XCircle } from '@phosphor-icons/react';
+import classNames from 'classnames/bind';
+import classes from './Alert.module.scss';
 
-import {
-  CheckCircle,
-  Info,
-  Warning,
-  WarningCircle,
-} from '@phosphor-icons/react'
-import classNames from 'classnames/bind'
-import { useEffect, useRef } from 'react'
+const cx = classNames.bind(classes);
 
-const cx = classNames.bind(classes)
-
-export type SeverityType = 'error' | 'warning' | 'info' | 'success'
+export type SeverityType = 'error' | 'warning' | 'info' | 'success';
 
 interface AlertProps {
-  severity: SeverityType
-  description?: string
-  title?: string
-  marginTop?: boolean
-  marginButton?: boolean
-  className?: string
-  isScrollIntoView?: boolean
+  severity: SeverityType;
+  description?: string;
+  title?: string;
+  marginTop?: boolean;
+  className?: string;
+  style?: React.CSSProperties;
+  onDismiss?: () => void;
 }
 
 export type AlertType = {
-  severity: SeverityType
-  title: string
-  description: string
-}
+  id: string;
+  severity: SeverityType;
+  title: string;
+  description: string;
+};
+
+const AlertsContext = createContext<any>(undefined);
+
+export const useAlerts = () => {
+  const context = useContext(AlertsContext);
+  if (!context) {
+    throw new Error('useAlerts must be used within an AlertsPromptProvider');
+  }
+  return context;
+};
 
 const getAlertIcon = (severity: SeverityType) => {
+  const iconProps = { size: 50 };
   switch (severity) {
+    case 'success':
+      return <CheckCircle {...iconProps} />;
     case 'error':
-      return <WarningCircle />
-    case 'warning':
-      return <Warning />
+      return <XCircle {...iconProps} />;
     case 'info':
-      return <Info />
-    case 'success':
-      return <CheckCircle />
+      return <Info {...iconProps} />;
+    case 'warning':
+      return <Warning {...iconProps} />;
+    default:
+      return <span style={{ fontSize: '5em' }}>❔</span>;
```

Comment:
Emojis can look different on each device and it's not ideal. Please use an icon from Phosphor or an SVG.

Reply:
Seems like you haven't replaced the emoji with an icon yet.